languages/ruby/lib/oso/polar/ffi/polar.rb

```diff
@@ -24,6 +24,7 @@ class Polar < ::FFI::AutoPointer
         # @raise [FFI::Error] if the FFI call returns an error.
         def self.create
           polar = Rust.new
+          Message.process_messages
```

Comment:
Doing this as a step in the init of another FFI type feels kinda weird. Are we expecting there to be waiting messages between allocating a new Polar instance and checking if that allocation resulted in an error?

Reply:
was in too many places, defined where they go in the readme now and only put them there